src/assets/typography.stories.mdx

```diff
@@ -0,0 +1,42 @@
+import { Meta, Story, Typeset, Source } from '@storybook/addon-docs/blocks';
+
+<Meta title="Typography" />
+
+export const typography = {
+  family: 'Neuzeit S LT Std',
+  weight: {
+    regular: '400',
+    bold: '800',
+  },
+  size: {
+    base: '17px'
```

Comment:
Is this not hard for you in the future of using `17px` as the base font size?
Because in normally we're using `16px` for that because it's easier to calculate it into the `rem` unit.

But yeah almost all of the components within our design are using `17px`, it might be easy for this situation of using `17px` as the base size. 🤔

What do you think?

Reply:
For now, I think maybe I leave it the same as the size in the design first. Reducing it looks a bit different.

https://user-images.githubusercontent.com/6965195/186322314-5d9b002d-b3b0-40a5-9be1-5c149bfd2a93.mov

With this font family, it seems to render fine 👍